rest/import_test.go

```diff
@@ -128,6 +128,89 @@ func TestXattrImportOldDoc(t *testing.T) {
 	assert.True(t, HasActiveChannel(rawDeleteResponse.Sync.Channels, "docDeleted"), "doc did not set _deleted:true for SDK delete")
 }
 
+// Test import of an SDK delete with ImportFilter .
```

Comment:
This test still appears to have a lot of content from TestXattrImportOldDoc.  I don't know if there's a connection between OldDoc handling in the sync function and _deleted:true presence in the import filter - was this included intentionally?

Reply:
It was included unintentionally, the unit has been revised by adding a doc via SDK, triggering SG import, deleting it via SDK, and finally triggering SG import to verify whether the deleted doc is getting imported.